Jenkinsfile

```diff
@@ -2,6 +2,6 @@
 DevelopPipeline(
     name: "ts_standardscripts",
     module_name: "lsst.ts.standardscripts",
-    extra_packages: ["lsst-ts/ts_observatory_control"],
+    extra_packages: ["lsst-ts/ts_cRIOpy", "lsst-ts/ts_observatory_control"],
```

Comment:
Should this be "lsst-ts/ts_criopy" instead? Not sure if this is case-sensitive here...

Reply:
No, this needs to be the name of the GitHub package, which is the `ts_cRIOpy` version.